lib/engine/g_1856/corporation.rb

```diff
@@ -15,7 +16,9 @@ def initialize(game, sym:, name:, **opts)
         @game = game
         @started = false
         @escrow = nil
+        @log = @game.log
```

Comment:
we shouldn't do this

Reply:
I changed `corporation.rb` to use `@game.log` instead in the remaining log statements

Reply:
what i meant by this was Corporation shouldn't have access to the game or the log